### PR TITLE
Add recommendations API

### PR DESCRIPTION
**Pull request by ShishirH**
Signed-off-by: Shishir Halaharvi <shihala1@in.ibm.com>

### CODE REVIEW

**Comment by ShishirH** [comment]
@dinogun Further changes required to be made after #50 and #74 are merged.
Please hold off merging this until then.

**Comment by ShishirH** [comment]
Added simple test that uses the new API in the tests folder, and made changes suggested for the application code. @chandrams @dinogun Please review and let me know if any other changes are required.

File: src/main/java/com/kruize/service/HealthService.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    private static int CURRENT_STATUS = STATUS_DOWN;
+
+    public static int getCurrentStatus()
+    {
+        return CURRENT_STATUS;
+    }
+
+    public static void setCurrentStatus(int currentStatus)
+    {
+        CURRENT_STATUS = currentStatus;
```
Comment (dinogun) [inline src/main/java/com/kruize/service/HealthService.java]: Can we have a check on the setter function please. In general all setters should validate what is being set.

File: tests/testAPI.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+AUTH_TOKEN=""
+
+#Check if jq exists
+if ! [ -x "$(command -v jq)" ]; then
+  echo "Error: jq is not installed."
+  exit 1
+fi
+
+#Check if Kruize is running without issues, and fetch recommendations generated
+while [[ $(curl -s -H "Authorization:Bearer $AUTH_TOKEN" "$KRUIZE_URL:$KRUIZE_PORT/health") == "UP" ]]; do
```
Comment (dinogun) [inline tests/testAPI.sh]: Suggest separating this out into functions, eg: one function that returns a list of all applications being monitored by Kruize, another that returns the info for a app that is passed as an arg, another that checks the validity of the AUTH TOKEN etc.

**Pull request merged**